src/main/java/uk/gov/companieshouse/acsp/manage/users/model/AcspMembersDao.java

```diff
@@ -6,14 +6,12 @@
 import org.springframework.data.annotation.CreatedDate;
 import org.springframework.data.annotation.Id;
 import org.springframework.data.annotation.Version;
-import org.springframework.data.mongodb.core.index.CompoundIndex;
 import org.springframework.data.mongodb.core.index.Indexed;
 import org.springframework.data.mongodb.core.mapping.Document;
 import org.springframework.data.mongodb.core.mapping.Field;
 import uk.gov.companieshouse.api.acsp_manage_users.model.AcspMembership.UserRoleEnum;
 
 @Document( "acsp_members" )
-@CompoundIndex( name = "acsp_user_idx", def = "{ 'acsp_number': 1, 'user_id': 1 }", unique = true )
```

Comment:
I've removed this because I don't think it's needed, and instead prohibits our intended functionality.   This was added originally when we were considering using upserts, however now we're only using inserts - if a member previously existed but was then removed, having this index would prevent a new document being inserted for them as the old one could still have the same `acsp_number` & `user_id` (just with `removed_by` set)...